### DISCUSSION TITLE
Can't show the name of conda environment

### DISCUSSION THREAD

**Discussion by YQisme** [JanDeDobbeleer/oh-my-posh#3418] [Troubleshoot]
When I `Open in Terminal` in a folder,everything is ok.
![image](https://user-images.githubusercontent.com/80308783/215284393-a8d9e377-8bcb-4cb4-9e16-b34bbf26a2a3.png)

But  when I `Open a new tab` in the Terminal,something happend.The name of conda environment can't show.

![image](https://user-images.githubusercontent.com/80308783/215284410-87f25361-8196-4492-9a5f-ae8e2dd53724.png)

**Comment by JanDeDobbeleer** [comment]
@YQisme this means the environment variables aren't passed in that session it seems. That's a question for Windows Terminal.

**Reply by YQisme** [reply]
> @YQisme this means the environment variables aren't passed in that session it seems. That's a question for Windows Terminal.

Is there any solution?

**Reply by JanDeDobbeleer** [reply]
I would ask the Windows Terminal team.

**Reply by YQisme** [reply]
> I would ask the Windows Terminal team.

ok, thank you very much!
